db.go

```diff
@@ -106,6 +111,33 @@ type Db interface {
 	// the database yet.
 	NewestSha() (sha *btcwire.ShaHash, height int64, err error)
 
+	// FetchAddrIndexTip returns the hash and block height of the most recent
+	// block which has had its address index populated. It will return
+	// ErrAddrIndexDoesNotExist along with a zero hash, and -1 if the
```

Comment:
Hmm yeah technically it's actually a pointer to a `btcwire.Shahash` which is itself the zero hash. Full response below.